### PR TITLE
Add installation troubleshooting page

### PR DESCRIPTION
**Pull request by ReToCode**
## Proposed Changes
- Add a page with common troubleshooting help

/cherrypick release-1.10

/cc @dprotaso, @psschwei, @skonto , @nak3 this should help us with the common issues we get asked in GH-issues and slack. The idea is that we can link to that page. In the future we can add common problems and their solutions as well. WDYT? Feedback from English native speaker also highly welcome :)

### CODE REVIEW

File: docs/install/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+NAME                                      READY   STATUS    RESTARTS   AGE
+activator-6b9dc4c9db-cl56b                1/1     Running   0          2m
+autoscaler-77f9b75856-f88qw               1/1     Running   0          2m
+controller-7dcb56fdb6-dbzrp               1/1     Running   0          2m
+domain-mapping-6bb8f95654-c575d           1/1     Running   0          2m
+domainmapping-webhook-c77dcfcfb-hg2wv     1/1     Running   0          2m
+webhook-78dc6ddddb-6868n                  1/1     Running   0          2m
+```
+
+**Knative Serving Networking Layer**
```
Comment (psschwei) [inline docs/install/troubleshooting.md:24]: nit: should we use tabs for each of the networking options, like we do on the [install page](https://github.com/knative/docs/blob/e10dc2982f7e4d7bff817c1e1a7b641247c375ec/docs/install/yaml-install/serving/install-serving-with-yaml.md#install-a-networking-layer)?

**Review by psschwei — COMMENTED** [review]
one minor nit, otherwise LGTM

File: docs/install/troubleshooting.md (hunk 1 above)
Comment (ReToCode) [reply docs/install/troubleshooting.md:24]: Good idea, updated.

**Review by psschwei — APPROVED** [review]
/lgtm
/approve

**Pull request merged**